### PR TITLE
About section

### PR DESCRIPTION
**Pull request by tomhebunn**
feat: added about section with info about act, speckle, methodology, data, and contributors. Made minor updates to the UI

### CODE REVIEW

**Review by Exilliar — COMMENTED** [review]
Looks really good! Some of those link buttons need the `src` attribute in them changing to `href` so that they act (the word, not acronym) as links. I've marked where the updates need to happen.

There's also something weird going on with the scroll thing being inside the page container, rather than in the normal place (so it's inside the page content rather than at the edge of the page). But I think that that might be a ARC problem, so not something to fix in this pr

File: src/views/About.vue [hunk 1]
```diff
@@ -1,34 +1,174 @@
...
+            ></iframe>
+          </v-card-actions>
+          <v-card-actions class="my-4">
+            <v-row dense>
+              <v-col cols='6' align='right'>
+                <v-btn color='primary' outlined src='https://speckle.systems/' target='_blank'>
+                  Speckle Home
+                </v-btn>
+              </v-col>
+              <v-col cols='6' align='left' scr='https://speckle.xyz/' target='_blank'>
```
[the remark this answers is not in the document]
Comment (tomhebunn) [reply src/views/About.vue]: done

File: src/views/About.vue [hunk 2]
```diff
@@ -1,34 +1,174 @@
...
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </v-card-actions>
+          <v-card-actions class="my-4">
+            <v-row dense>
+              <v-col cols='6' align='right'>
+                <v-btn color='primary' outlined src='https://speckle.systems/' target='_blank'>
```
[the remark this answers is not in the document]
Comment (tomhebunn) [reply src/views/About.vue]: done

File: src/views/About.vue [hunk 3]
```diff
@@ -1,34 +1,174 @@
...
+          <v-card-text>
+            The data within <strong>act</strong> has been segregated by region. Carbon factors have been selected from the following, openly available, 
+            sources...
+          </v-card-text>
+          <v-card-actions class="mx-3 mb-4">
+            <v-row>
+              <v-col v-for='geo in dataSources' :key='geo.name' cols='12'>
+                <div><strong>{{ geo.name }}</strong></div>
+                <div v-for='source in geo.sources' :key='source.name'>
+                  <v-btn outlined small :src='source.url' class='my-1' color='secondary' target="_blank">
```
[the remark this answers is not in the document]
Comment (tomhebunn) [reply src/views/About.vue]: done

File: src/views/About.vue [hunk 4]
```diff
@@ -1,34 +1,174 @@
...
+                    rounded
+                    class='my-1'
+                    height="100%"
+                  >
+                    <v-card flat height='100%' class='card-outter'>
+                      <v-img :src="item.img" aspect-ratio="1" contain max-height='240px'></v-img>
+                      <v-card-title>{{ item.name }}</v-card-title>
+                      <v-card-subtitle>{{ item.byline }}</v-card-subtitle>
+                      <v-card-actions fill-height class='card-actions'>
+                        <v-btn icon color='primary' class="ma-2" :disabled="item.git===''" :src="item.git">
```
[the remark this answers is not in the document]
Comment (tomhebunn) [reply src/views/About.vue]: done

File: src/views/About.vue [hunk 5]
```diff
@@ -1,34 +1,174 @@
...
+                          <v-icon>
+                            mdi-github
+                          </v-icon>
+                        </v-btn>
+                        <v-btn icon color='primary' class="ma-2" :disabled="item.linkedin===''">
```
[the remark this answers is not in the document]
Comment (tomhebunn) [reply src/views/About.vue]: done. also added target="_blank" to open in new tab

**Review by Exilliar — APPROVED** [review]
Good to merge!

I think that there's a future task to make the popup on the landing page only appear on the first time that the user goes onto the site. But that can be done in the future.

**Pull request merged**